ocs_ci/ocs/platform_nodes.py

```diff
@@ -2424,5 +2423,129 @@ def restart_nodes_by_stop_and_start_teardown(self):
         if len(worker_nodes_not_ready) > 0:
             for not_ready_node in worker_nodes_not_ready:
                 cmd = f"ibmcloud ks worker reboot --cluster {cluster_id} --worker {not_ready_node} -f"
-                out = run_cmd(cmd)
+                out = run_ibmcloud_cmd(cmd)
                 logger.info(f"Node restart command output: {out}")
+
+    def check_workers_ready_state(self, cmd):
```

Comment:
Just a minor nit which can be addressed in another PR otherwise LGTM. `cmd` argument here looks like any generic command can be passed, where as we are specifically checking worker ready state, hence we can omit this argument and make it as a fixed variable in the fucntion itself.